### DISCUSSION TITLE
Deckgl arc_layer: How to change the arc direction on the world map?

### DISCUSSION THREAD

**Discussion by kforris** [visgl/deck.gl#6458] [Q&A] [answered]
Hi Community, im currently working on a personal project presenting packages shipped from Toronto to China. 

My plan was:
- to use Deck.gl ArcLayer to show the shipment trajectory, which usually came the same as flights trajectory since they go with air.
usually, they go like this: (Across the Pacific Ocean)
![image](https://user-images.githubusercontent.com/58551353/144212809-a23e28ae-41f8-47dd-aa0c-b2dcf1923b26.png)

However,  my results is showing like this: 
![image](https://user-images.githubusercontent.com/58551353/144212918-8ac0776c-2e0c-4dba-84fe-4a29f3a1bf3c.png)

I kinda assumed it was because the Pacific Ocean isnt the center of world map, which represented by Mapbox layer.
or, deck gl just shows the closest distance between two points on a 2d dimension level.

Please let me know if there's any solution to this.
Much appreciated!

**Comment by Pessimistress** [comment, marked as answer]
Use [wrapLongitude](https://deck.gl/docs/api-reference/core/layer#wraplongitude)


```js
new Deck({
  ...
  views: new MapView({repeat: true}),
  layers: [
    new ArcLayer({
      ...
      wrapLongitude: true
    })
  ]
})
```

**Reply by kforris** [reply]
Much appreciated! Works like a charm!
